base/genericmemory.jl

```diff
@@ -3,9 +3,21 @@
 ## genericmemory.jl: Managed Memory
 
 """
-    GenericMemory{kind::Symbol, T, addrspace=Core.CPU} <: AbstractVector{T}
+    GenericMemory{kind::Symbol, T, addrspace=Core.CPU} <: DenseVector{T}
 
-One-dimensional dense array with elements of type `T`.
+Fixed-size [`DenseVector{T}`](@ref DenseVector).
```

Comment:
```suggestion
A [`DenseVector{T}`](@ref DenseVector). Fixed-size, in the sense that the length of a value of this type doesn't change after construction.
```

Maybe it'd be better to be more precise regarding "fixed-size"?

Reply:
I don't really see this is clearer. Is there an ambiguity with the original version? Would "constant length" be clearer?

Reply:
Perhaps the current version is OK. My motivation is to avoid confusion with the StaticArrays.jl-sense of "constant length".